### ISSUE TITLE
Please add omitPrismicScript option to documentation

### ISSUE THREAD

**Issue by chadfaurie** [birkir/gatsby-source-prismic-graphql#119]
I had been struggling for a while with lighthouse audits and found 'omitPrismicScript' in the code, which cleared up a some blocking resources since I am not going to use previews in production

It would be beneficial to have this is in the documentation

**Comment by erictaylor** [comment]
I agree. Additionally it feels that if `previews: false` then the scripts shouldn't be included anyways.
